packages/@webex/plugin-cc/src/services/core/aqm-reqs.ts

```diff
@@ -2,6 +2,7 @@ import {Signal} from './Signal'; // TODO: remove and use Event Emitters after Ad
 import {Msg} from './GlobalTypes';
 import * as Err from './Err';
 import {HTTP_METHODS, WebexRequestPayload} from '../../types';
+import {WebSocketManager} from './WebSocket/WebSocketManager';
```

Comment:
The changes in this file are simply replacing the web socket manager in the same manner in which we have `aqm-notfifs` inside of the the agent desktop.
It uses signals as well (logic is basically the same as agent desktop, with minor refactoring).